### PR TITLE
[guide] Add a basic ModifyGuide

### PR DESCRIPTION
**Pull request by thejud**
Per issue https://github.com/saulpw/visidata/issues/2313 Add a new ModifyGuide

### CODE REVIEW

File: visidata/guides/ModifyGuide.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+## Edit rows
+
+- {help.commands.sysedit-selected}
+
+- {help.commands.edit_cell}
+
+While editing a cell, 
+    - [:keys]Ctrl+o[/] to edit the cell in a text editor.
```
Comment (saulpw) [inline visidata/guides/ModifyGuide.md]: For better or worse, we've standardized on `Ctrl+O` (capital O).

File: visidata/guides/ModifyGuide.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - [:code]Tab[/] to finish editing and go to the next cell
+    - [:code]Shift+Tab[/] to finish editing and go to the previous cell
+
+Modifying a sheet displays a `[M]` in the lower right status.
+
+- {help.options.quitguard}
+
+## Deferred Sheets
+
+Some sheet types support *deferred* saving. Examples include **SQLiteSheet** and **Dir Sheet**. Modifications are colored: green for added rows, red for deleted rows, and yellow for modified cells.
```
Comment (saulpw) [inline visidata/guides/ModifyGuide.md]: They are colored according to their options.  Use `[:add_pending]` and change_pending and delete_pending at least to color the given text here, and maybe called out literally in a list below, like we do with other options, depending on if we want to start including all the interface options (which might be nice).

**Review by saulpw — APPROVED** [review]

File: visidata/guides/ModifyGuide.md (hunk 2 above)
Comment (anjakefala) [reply visidata/guides/ModifyGuide.md]: I forgot about those options! Good call.

**Comment by anjakefala** [comment]
Screenshot from 2024-09-29 21-01-50

**Comment by anjakefala** [comment]
Thanks so much @thejud! I added some options and additional info about Deferred Sheets. =)

**Pull request merged**